src/garage/sampler/batch_sampler.py

```diff
@@ -12,8 +12,9 @@ class BatchSampler(BaseSampler):
 
     """
 
-    def __init__(self, algo, env):
-        super(BatchSampler, self).__init__(algo, env)
+    def __init__(self, algo, env, n_envs):
+        super().__init__(algo, env)
+        self.n_envs = n_envs
```

Comment:
This variable does not seem to be used?

Reply:
Ohh yes, updated. Thanks for pointing out.